### ISSUE TITLE
Blink crashes switches theme to white background despite dark theme still being indicated as selected

### ISSUE THREAD

**Issue by jaclu** [blinksh/blink#1342]
I have noticed this for almost a year and every time I suddenly get a shining white screen in my face on any new terminal I scream in pain since for dark themes I tend to dial up the brightness :(

Not until now have I been systematic enough to be able to give what at lest for me is a simple and repeatable pattern.
If I run du as is I get the expected result, but with sizes that aren’t really saying much. If I do the normal ``` du -sh ``` that terminal hangs. When I kill it and start a new terminal, or terminate Signal and restart it I get the white screen, or rather I get a background that is the default color theme from iOS, so at night time it just flashes white for a split second before going black again, same flash each time I spawn a new terminal.  But in either case if I go into Appearance, the selected dark theme is still ticked.
This happens both on my phone and iPad after crashes.  Since my actual selection is still marked as selected I first have to select something else and then select my preference again to get things back to the intended state again.

I have uninstalled and reinstalled on both, a couple of times and tried both without and with iCloud sync of settings, so that is probably not the cause of this.

This happens for me any time Blink crashes, the above is just the first example of a simple action that triggers a crash every time. So getting du fixed would also be nice, but at least for me it is way down on the list, I hadn’t even used it until today when checking for large files related to another bug. Turning the iPad into a light bomb is really bothersome when you use it a lot :)

So cheers for a really good terminal app! But plz let me stay on the dark side :)

**Comment by jaclu** [comment]
How come you closed it? The issue with the screen going white is long gone, so sure that part is no longer an issue. So far all good.

But the sample test that I mentioned that blink hangs forever if you do du -sh is still unresolved

**Comment by carloscabanero** [comment]
Oh, understood. Main issue here was the problem with the themes which is why we closed as complete. I did not notice there was another additional issue with `du`. Let's create a separate issue for that so we can categorize that properly, and reference here.

**Comment by jaclu** [comment]
will do
